File: 169-majority-element/169-majority-element.py
class Solution:
    def majorityElement(self, nums: List[int]) -> int:
        n=set(nums)
        count=0
        d={}
        for i in n:
            time=nums.count(i)
            d.update({i:time})   
        Keymax = max(d, key= lambda x: d[x])
        return Keymax
        

File: 169-majority-element/169-majority-element.java
class Solution {
    public int majorityElement(int[] nums) {
        int []arr=new int [50001];
        Arrays.fill(arr, 0);
        for(int i=0;i<nums.length;i++){
            if(nums[i]>0){
                arr[nums[i]]++;
            }
        }
        int a=0;
        int max=0;
        for(int i=0;i<50001;i++){
            if(max<arr[i]){
                max=arr[i];
                a=i;
            }
        }
        
        return a;
    }
}